index.html

```diff
@@ -52,20 +55,30 @@ <h3>Mobile management</h3>
 
   <div class="jumbotron theme-default">
     <div class="container">
-      <h3>Easy and simple</h3>
-      <p>
-        Flyve MDM counts with an intuitive and smooth interface which integrated with GLPI, facilitates the usage, specially for users who implement this IT and asset management software.
-      </p>
+      <div class="col-md-12 col-md-push-12 text-center">
+        <img class="img-responsive icon visible-xs-inline-block visible-sm-inline-block visible-md-inline-block visible-lg-inline-block visible-xl-inline-block"
+          alt="Flyve" src="images/easy-icon.svg" data-source-index="2">
+      </div>
+
+      <div class="col-md-12 col-md-pull-12">
+        <h3><b>Easy and simple</b></h3>
+        <p class="type-p2">
+          Flyve MDM counts with an intuitive and smooth interface which <br/>
```

Comment:
Add `<br>` is bad praxis.
May you should add padding or margin to the Paragraph element.

Reply:
No problem, I'll work on it.